content/blog/2020-08-19-train-a-sequence-model-with-poutyne/index.en.md

```diff
@@ -420,12 +440,12 @@ good for a first model. Also, we can see that our model did not seem to have ove
 ## Bigger model
 
 It seems like our model performed pretty well, but just for fun, let's unleash the full potential of LSTM using a
-bidirectional approach (bi-LSTM). What it means is that instead of _simply_ viewing the sequence from the start to the end, we
+bidirectional approach (bidirectional LSTM). What it means is that instead of _simply_ viewing the sequence from the start to the end, we
 also train the model to see the sequence from the end to the start. It's important to state that the two directions are
 not shared, meaning that we _see_ the sequence in one direction at the time, but we gather the information from both directions into the 
 fully connected layer. That way, our model can get insight from both directions.
 
-Instead of using only one layer, let's use a bi-LSTM, which means that we use two layers of hidden state.
+Instead of using only one layer, let's use a bidirectional bi-LSTM, which means that we use two layers of hidden state for each direction.
```

Comment:
```suggestion
Instead of using only one layer, let's use a bidirectional LSTM, which means that we use two layers of hidden state for each direction.
```

Reply:
Non, j'avais fait une erreur, un bi-lstm signifie juste deux layers donc c'est un bidirectional bi-lstm.